cases/models.py

```diff
@@ -153,6 +153,9 @@ class Case(AbstractModel):
         ("?", "Don’t know"),
     ]
 
+    # State
+    closed = models.BooleanField(default=False)
```

Comment:
Is this ever likely to be a non-binary state (eg like FMS reports), or is that a future problem?

Reply:
Nothing in the current proposed work appears to suggest any other sort of state changes (there's a 'priority' ticket, but that wouldn't be a state (though I guess you'd not have a closed priority one, would you, but anyway, yes think can come back if need be in future)).